pkgs/top-level/all-packages.nix

```diff
@@ -8641,9 +8641,9 @@ with pkgs;
 
   neuron-notes = haskell.lib.compose.justStaticExecutables (haskell.lib.compose.generateOptparseApplicativeCompletion "neuron" haskellPackages.neuron);
 
-  ngrok = ngrok-2;
+  ngrok = ngrok-3;
 
-  ngrok-2 = callPackage ../tools/networking/ngrok-2 { };
+  ngrok-3 = callPackage ../tools/networking/ngrok-3 { };
```

Comment:
We can simply this and remove the versioned attr

Reply:
Arg, https://github.com/NixOS/nixpkgs/pull/175225 was merged only 15 days ago. That's why it can now remove the versioning, because now there only a single version in the repo.

Reply:
Oh, unfortunate timing. Done.